core/code/portal_detail.js

```diff
@@ -33,7 +33,7 @@ var handleResponse = function(deferred, guid, data, success) {
 
   if (success) {
 
-    var dict = decodeArray.portalDetail(data.result);
+    var dict = decodeArray.portal(data.result, 'detailed');
```

Comment:
note: `data.result` is parsed a second time with 
```window.runHooks ('portalDetailLoaded', {guid:guid, success:success, details:dict, ent:ent});```

Reply:
Should we add some remark to sources?

Reply:
I suggest to reverse the order of the code here:
creating an entity first and then , use the the parsed data to populate `portalDetails`'s cache
like in this commit https://github.com/le-jeu/ingress-intel-total-conversion/commit/bf916d18c6aa3406cd4409fce58432215e772c5e (forgetting the specificity of this branch)

this can be in this PR or in a later one

Reply:
I prefer later.